lib/primitives/base-element.js

```diff
@@ -35,6 +35,103 @@ module.exports = function(registry) {
       return this.content;
     },
 
+    /// Finds the given elements in the element tree
+    /// Returns ArrayElement of elements or null
+    findRecursive: function() {
+      var elementNames = [].concat.apply([], arguments);
+      var ArrayElement = registry.getElementClass('array');
+      var array = new ArrayElement();
+      var parents;
+
+      if (elementNames[elementNames.length - 1].element === 'array') {
```

Comment:
No, because `"random string".element` returns undefined and this the comparison fails. If that wasn't the case then all the tests that call `findRecursive` would fail.